test/conformance/service_test.go

```diff
@@ -334,18 +355,40 @@ func TestReleaseService(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Failed to create initial Service %v: %v", names.Service, err)
 	}
+
+	// This is just creating the service, so make sure it ready.
+	logger.Info("Waiting for Service to transition to Ready.")
+	if err := test.WaitForServiceState(clients.ServingClient, names.Service, test.IsServiceReady, "Verify Service Initial State"); err != nil {
```

Comment:
CreateReleaseServiceWithLatest already waits for the Service to be Ready. This is a noop.

Reply:
Done.